topics.md

```diff
@@ -274,3 +276,68 @@ additional labels to tag the k8s resources created during deployment
     example1: "some-value"
     example2: "some-value"
 ```
+
+## Resource Limits and Quotas
+
+All the pods created by the operator are set with with a resources section to their spec, so it is possible to apply a ResourceQuota on the namespace of the Redis Enterprise Cluster. The operator itself is set with resources limit and request.
+The recommended settings are set in the operator.yaml file and the bundles. The operator was tested and proved to be working in minimal workloads with the following settings in operator.yaml:
+
+
+```yaml
+  resources:
+    limits:
+      cpu: 0.5
+      memory: 256Mi
+    requests:
+      cpu: 0.5
+      memory: 256Mi
+```
+
+When creating ResourceQuota, be careful when applying quotas on ConfigMaps. When testing the operator the limit was found to be met even when one ConfigMap was used, perhaps due to enforcement logic of some sort. The following ResourceQuota worked on internal testing, but might need tweaking according to the deployment scenario:
+```yaml
+  hard:
+    secrets: "40"
+    persistentvolumeclaims: "20"
+    replicationcontrollers: "40"
+    pods: "40"
+    requests.storage: "120400Mi"
+    services: "20"
+    requests.memory: "43344Mi"
+    limits.memory: "57792Mi"
+    limits.cpu: "64"
+    requests.cpu: "48"
+```
+
+## Custom Resource Deletion
+### REDB Deletion
+The Redis Enterprise Database (REDB) object has a finalizer, to make sure the database is deleted before the REDB custom resource is removed from k8s.  
+The finalizer name is `finalizer.redisenterprisedatabases.app.redislabs.com`.  
+When a user requests the deletion of REDB (for example by running `kubectl delete redb <name>`), the following happens:
+1. K8s API adds `DeletionTimestamp` to the REDB resource.
+2. The Operator notices the `DeletionTimestamp`, and sends delete request to the RS API.
+3. When RS API approves delete request, the operator removes the REDB finalizer.
+4. K8s cleans up the REDB resource, now that it has no finalizers.
+
+If for some reason the user ends up with a REDB resource that can't be deleted, because the finalizer can't be removed, they can remove the finalizer manually by editing the REDB resource.
```

Comment:
```suggestion
If for some reason the user ends up with an REDB resource that can't be deleted, because the finalizer can't be removed, they can remove the finalizer manually by editing the REDB resource.
```